### PR TITLE
Fixes #1445:Use of icon instead of starred

### LINKED ISSUE #1445: Instead of "Starred" use icons
Instead of using show "starred" use icons:
* To show all starred sessions in the UI show a filled black icon
* To show all sessions in the UI show an icon which is unfilled
ui-39-512

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1445 

Changes: Added star icon and changed javascript in schedule.hbs

Link for reviewing-
https://agbilotia1998.github.io/FOSSASIA-SUMMIT/schedule.html

### CODE REVIEW

**Comment by geekyd** [comment]
@agbilotia1998 Great work!
I think it would be nice if we use a common design pattern. 

**When the button is not active**
screenshot from 2017-08-19 15-08-54

**When the button is active**
screenshot from 2017-08-19 15-11-19

**Comment by agbilotia1998** [comment]
Okay @geekyd ,I will make the changes.Thanks.

**Comment by agbilotia1998** [comment]
@geekyd please review now.
Test link to review https://agbilotia1998.github.io/FOSSASIA-SUMMIT/schedule.html

**Comment by geekyd** [comment]
@agbilotia1998 Can you please update the branch there were changes made to the buttons layout in #1442, also update the deployment link :)

**Comment by agbilotia1998** [comment]
@geekyd  branch has been updated , other than this whic deployment link should be updated?

**Comment by Princu7** [comment]
@agbilotia1998 Update the GitHub Page deployment link with the new changes. Thanks!

**Comment by agbilotia1998** [comment]
@geekyd @Princu7 deployment link is updated , please check.

**Comment by Princu7** [comment]
@agb Thanks for updating. But there seems to one issue. I can still see the ics download button on the upper right of the page. It shouldn't be there since we have combined the png download and the ics download into a single dropdown button already. It is also present in this commit. Please fix it. Thanks :)

* We have two download button for ics. It shouldn't be there.
664e0b16-64bc-4a97-8a51-fca7001d414a

* Please compare to this (There is noextra ics download button)
a6114ef1-1fd8-4792-b28a-51702dfe0c19

**Comment by agbilotia1998** [comment]
@Princu7 done please review.

**Comment by agbilotia1998** [comment]
@Princu7 I think gh-pages are not getting updated, I have done the changes.
screenshot from 2017-08-20 17-23-36

**Comment by Princu7** [comment]
@agbilotia1998 Yeah, it is working now! Thanks!

**Comment by Princu7** [comment]
One small suggestion you can do is instead of using javascript to set the color and background color of the button, add some classes with that properties and then toggle those classes on click. It would look cleaner that way I guess.

**Comment by agbilotia1998** [comment]
Okay @Princu7 I will do the changes.

**Comment by agbilotia1998** [comment]
When adding properties to css class and toggling the classes, bootstrap is overriding and using !important is not a good practice.Is there any other way?

**Comment by Princu7** [comment]
@agbilotia1998 I will check and get back to you. Thanks!

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1868,8 +1868,8 @@ a.skip:hover {
 // style from tracks
 
 #starred {
-  background-color:$white-gray;
   color:black;
+  background-color:#eee;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Please add .scss variable, don't hardcode variables.

**Comment by agbilotia1998** [comment]
@aayusharora done please review.

**Pull request merged**